components/Cards/DetailsCard.js

```diff
@@ -448,8 +490,10 @@ export default function DetailsCard({
             )}
         </StyledCardBody>
       </StyledCardArticle>
-      <FormComments fetchComments={fetchComments} />
-      {comments.length > 0 && (
+      {!isEditingPlaceId ? (
+        <FormComments fetchComments={fetchComments} />
+      ) : null}
```

Comment:
Just an advise:
Render the `FormComments` component only if isEditingPlaceId is `false`

```suggestion
     {!isEditingPlaceId && <FormComments fetchComments={fetchComments} />}
```